packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -1031,10 +1031,6 @@ export default class TradeStore extends BaseStore {
 
     @action.bound
     onUnmount() {
-        if (this.should_skip_prepost_lifecycle) {
```

Comment:
I don't think we need to remove this block as well. This is to stop unmount from happening when visiting `contract details` page from trade page.

Reply:
Correct. I just used the code snippet you sent.